src/engine/MaxConstraint.h

```diff
@@ -125,6 +125,7 @@ class MaxConstraint : public PiecewiseLinearConstraint
  private:
     unsigned _f;
     Set<unsigned> _elements;
+    double _score;
```

Comment:
If you define _score in the parent class, I don't think you can/should define it in the children without. I think this will mask the parent definition.

Reply:
Removed _score from children classes.